subsys/bluetooth/audio/has.c

```diff
@@ -100,6 +102,10 @@ static struct has_client {
 #endif /* CONFIG_BT_EATT */
 	} params;
 
+	struct  {
+		bool in_progress;
+		uint8_t preset_changed_index_next;
+	} ntf_bonded;
```

Comment:
Does HAS mandate bonding, or can HAS be used without bonding? i.e. should this be guarded by `BT_BONDABLE`?

Reply:
It can be used without bonding, however the characteristic value notifications of the values that changed shall be sent to bonded devices only once reconnected

Reply:
Does it then make sense to have this functionality if `CONFIG_BT_BONDABLE=n`?